src/layout/Tabbed.js

```diff
@@ -69,6 +69,14 @@
         HTMLWidget.prototype.update.apply(this, arguments);
         var context = this;
 
+        if (this._tabContainer) {this._tabContainer.remove();}
+        if (this.tabsOnBottom()) {
+            this._tabContainer = element.append("div");
```

Comment:
I would set the class to match the tabLocation as per my comment above (note:  you will need to use attr("class", this.tabLocation())